validation/td-validation.ttl

```diff
@@ -367,15 +367,31 @@ well-known types necessary to implement the WoT interaction model described in
 operations types SHOULD NOT be arbitrarily set by servients
 </span>.
 </p>
+
+<p>
+The optional <code>response</code> name-value pair can be used to provide metadata, such as content type information, that is only itended for the reponse messages. 
```

Comment:
can be used to provide metadata for the expected response message.
With the core Vocabulary, it only includes content type information,
but TD Context Extensions could be applied.